File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q35.py
def main():
    #Input
    num = int(input("Enter a 4 digits number: "))

    #Processing

    first_digit = round(num / 1000)
    second_digit =round((num % 1000)/100)
    third_digit = round(((num % 1000) % 100) / 10)
    fourth_digit = ((num % 1000) % 100) % 10
    
    sum_result = first_digit + second_digit + third_digit + fourth_digit 

    #Output
    print(f"The result of the sum of the digits from entered number({num}) is equal to: {sum_result}")


main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q35.js
import {question} from "readline-sync"
function main(){
    //Input
    const num = parseInt(question("Enter a 4 digits number: ")) 

    //Processing
    const first_number = Math.floor(num / 1000) 
    const second_number = Math.floor((num % 1000)/100)
    const third_number = Math.floor((((num % 1000) % 100))/10)
    const fourth_number = ((num % 1000) % 100) % 10

    const sum_result = first_number + second_number + third_number + fourth_number

    //Output
    console.log(`The result of the sum between the number of ${num} (${first_number}, ${second_number}, ${third_number} e ${fourth_number}) is equal to: ${result_sum}`)

}

main()